src/menu/menu.scss

```diff
@@ -149,6 +150,32 @@ $iui-active-outline: thin solid t(iui-color-foreground-primary);
       }
     }
   }
+
+  &-skeleton {
+    --iui-menu-item-skeleton-max-width: 30%;
+    cursor: auto;
+
+    .iui-content {
+      max-width: var(--iui-menu-item-skeleton-max-width);
+    }
+
+    .iui-menu-label {
+      width: 100%;
+      height: $iui-line-height * 0.7;
```

Comment:
Full line height doesn't looks nice as it takes full height so I just to the nicest looking multiplier.

Reply:
I tested it using our font sizes, I like this approach.
![Screen Shot 2022-04-07 at 11 36 46 AM](https://user-images.githubusercontent.com/849817/162237653-402ef687-8699-4293-814e-db5d134f76d9.png)

Reply:
Thanks for the suggestion! Added.